api/v1/nodefeaturediscovery_types.go

```diff
@@ -87,8 +87,14 @@ type OperandSpec struct {
 
 	// ServicePort specifies the TCP port that nfd-master
 	// listens for incoming requests.
+	// DEPRECATED: gRPC is being deprecated in favor of the NodeFeature API
+	// This will be removed in a future release
```

Comment:
nit: we might want make this deprecation in a separate PR to make it more explicit and clearer